event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/service/EventsService.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.services.eventsourcing.source.api.service;
+
+import static java.util.stream.Collectors.toList;
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.Direction.BACKWARD;
+import static uk.gov.justice.services.eventsourcing.repository.jdbc.Direction.FORWARD;
+
+import uk.gov.justice.services.common.converter.StringToJsonObjectConverter;
+import uk.gov.justice.services.common.converter.ZonedDateTimes;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.Direction;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.Position;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.Event;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventJdbcRepository;
+import uk.gov.justice.services.eventsourcing.source.api.feed.event.EventEntry;
+
+import java.sql.SQLException;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Function;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.json.JsonObject;
+
+@ApplicationScoped
+public class EventsService {
+
+    @Inject
+    private EventJdbcRepository repository;
+
+    public List<EventEntry> events(final Position position,
+                                   final Direction direction,
+                                   final long pageSize,
+                                   final UUID streamId) throws SQLException {
+
+        final List<Event> entities = entitiesPage(position, direction, pageSize, streamId);
```

Comment:
I'm not sure about some of the method naming, it doesn't seem that fluent,

maybe the variable should also be something like entitiesPage or pageOfEntities ?
Again i'd prefer the parameter ordering to have the most significant item at the front, keeping consistent with before (i.e the streamId first, pageSize is last)

Reply:
corrected